src/TableContent.js

```diff
@@ -24,16 +33,19 @@ const TableContent = ({tableHeader, searchedColumnIndex, searchQuery, orderedCol
             enableDownload && <Table.TextHeaderCell className={`downloadHeader`} flexBasis={100} flexShrink={100} flexGrow={100}>
               {
                 checkedRows.length > 0 ?
-                  <a href={URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()}>
-                    Download {checkedRows.length} {checkedRows.length === 1 ? `entry` : `entries`}
-                  </a> :
-                  `Download`
+                  <div>
```

Comment:
I think the conditional rendering could be improved here, so you don't end up with duplication.